libvmi/os/windows/core.c

```diff
@@ -84,24 +86,24 @@ win_ver_t pe2version(vmi_instance_t vmi, addr_t kernbase_pa)
 
     switch (optional_header_type) {
         case IMAGE_PE32_MAGIC:
-            major_os_version=oh32->major_os_version;
-            minor_os_version=oh32->minor_os_version;
+            *major=oh32->major_linker_version;
+            *minor=oh32->minor_linker_version;
             break;
         case IMAGE_PE32_PLUS_MAGIC:
-            major_os_version=oh32plus->major_os_version;
-            minor_os_version=oh32plus->minor_os_version;
+            *major=oh32plus->major_linker_version;
+            *minor=oh32plus->minor_linker_version;
```

Comment:
This subtle change from `major_os_version` to `major_linker_version` is causing breakage for Windows 10 VMs